vendors/halo2/src/bn254_blake2b_writer.cc

```diff
@@ -1,13 +1,60 @@
 #include "vendors/halo2/include/bn254_blake2b_writer.h"
 
```

Comment:
It seems `bn254_blake2b_writer.h` doesn't include <stddef.h>. Could you add it for `size_t`?